### DISCUSSION TITLE
How can I make my repos to become popular ?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#136563] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I promote my works on GitHub ?

**Comment by ferid333** [comment]
To promote your work on GitHub:

1. **Create a Strong README:** Make sure each of your projects has a well-written README that explains what it does, how to use it, and why it’s useful.

2. **Contribute to Open Source:** Get involved in other open-source projects. This helps you connect with other developers and increases visibility for your own projects.

3. **Share on Social Media:** Post about your projects on platforms like Twitter, LinkedIn, or Reddit. Use relevant hashtags and communities to reach a broader audience.

4. **Join GitHub Discussions:** Participate in GitHub Discussions and share your work when it’s relevant to the conversation.

5. **Write Blog Posts:** Create tutorials or blog posts about your projects or related topics, and link back to your GitHub repository.

6. **Engage with the Community:** Comment on issues, participate in discussions, and collaborate with others on GitHub. Building relationships can help get your work noticed.

These strategies can help you build an audience and get more attention for your projects on GitHub.

**Reply by ghost** [reply]
Thanks
